requirements_prod.txt

```diff
@@ -3,4 +3,5 @@ lxml
 psycopg2-binary
 requests-file==1.5.1
 requests-oauthlib
+retry
```

Comment:
It'd be nice to pin dependencies with version.

Reply:
I don't think this one is explicitly in the `constraints.txt` file for Airflow, so I'll go ahead and do that!